### DISCUSSION TITLE
Put the ControlNet models inside the models/ControlNet folder.

### DISCUSSION THREAD

**Discussion by philmetzger** [Mikubill/sd-webui-controlnet#349] [Q&A] [answered]
Hey little confused on this usage step:
`Put the ControlNet models (.pt, .pth, .ckpt or .safetensors) inside the models/ControlNet folder.`

Where do I get those files? In https://github.com/Mikubill/sd-webui-controlnet/tree/main/models there are only yaml files.

**Comment by philmetzger** [comment, marked as answer]
Ah from here: https://huggingface.co/lllyasviel/ControlNet/tree/main/models

**Reply by brunogcar** [reply]
smaller sizes and actually all the models you would need including pruned of https://huggingface.co/TencentARC/T2I-Adapter/tree/main/models as well 

https://huggingface.co/webui/ControlNet-modules-safetensors/tree/main
